traffic_ops/app/lib/Utils/Tenant.pm

```diff
@@ -64,6 +64,10 @@ sub new {
         ->get_column('value')->single();
     my $ignore_tenancy = defined($ignore_tenancy_value) ? $ignore_tenancy_value : 0;
 
+    my $ignore_ds_tmuser_assignment_value = $dbh->resultset("Parameter")->search( { config_file => 'global', name => 'ignore-ds-tmuser-assignment' } )
```

Comment:
i would really prefer if we can get rid of the 'ignore-ds-tmuser-assignment' parameter and use only the 'ignore-tenancy'. is that possible?